frontend/src/views/main/project/InspectorWrapper.vue

```diff
@@ -254,9 +258,10 @@ function next() {
 }
 
 watch(() => inputData.value, async (newValue, oldValue) => {
-  console.log("RowNb changed");
-  const pushStore = usePushStore();
-  await pushStore.fetchPushSuggestions(inspection.value!.model.id, inspection.value!.dataset.id, rowNb.value, newValue, modelFeatures.value);
+  if (newValue !== undefined) {
+    const pushStore = usePushStore();
+    await pushStore.fetchPushSuggestions(inspection.value!.model.id, inspection.value!.dataset.id, rowNb.value, newValue, modelFeatures.value);
```

Comment:
what does !. do ?

Reply:
It's a "non null" operator. It'll check if the previous variable is null, and if so, return false directly. It allows chaining by doing

`if (value!.othervalue!.lastvalue)` in a single line rather than 3 separate checks

Reply:
👍